### PR TITLE
Feature/openrc gentoo

### PR DESCRIPTION
**Pull request by bodsch**
here comes an systemd less version of gentoo :)
i personally prefer a system with openrc

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -14,15 +14,21 @@ jobs:
       fail-fast: false
       matrix:
         distribution:
-          - image: alpine
+          - image: gentoo-openrc
             version: "latest"
-            platform: "linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64/v8"
+            platform: "linux/amd64"
           - image: gentoo
             version: "latest"
```
Comment (diodonfrost) [inline .github/workflows/build.yml:21]: Openrc is the default init system of Gentoo, You can replace gentoo-systemd image by Gentoo openrc. Just keep the version to latest.

File: .github/workflows/push.yml [hunk 2]
```diff
@@ -15,9 +15,9 @@ jobs:
       fail-fast: false
       matrix:
         distribution:
-          - image: alpine
+          - image: gentoo-openrc
             version: "latest"
-            platform: "linux/386,linux/amd64,linux/arm/v6,linux/arm/v7,linux/arm64/v8"
+            platform: "linux/amd64"
```
Comment (diodonfrost) [inline .github/workflows/push.yml:20]: same as above

**Review by diodonfrost — CHANGES_REQUESTED** [review]
I think you can replace the Gentoo systemd image by the Gentoo openrc.

**Comment by bodsch** [comment]
> I think you can replace the Gentoo systemd image by the Gentoo openrc.

You are right!
I could have thought of it myself. :)
My wish was to have everything from one source if possible, that makes automated testing much easier. ;)
The PR can be closed.

**Pull request closed without merging**